raiden/raiden_event_handler.py

```diff
@@ -224,7 +224,14 @@ def handle_missing_metadata_events(
     ) -> None:
         new_events: List[Event] = []
         for wrapped_event in event.dependant_events:
-            metadata = fetch_address_metadata(wrapped_event.recipient)
+            try:
+                metadata = query_address_metadata(raiden.pfs_config, wrapped_event.recipient)
+            except ServiceRequestFailed:
+                log.error(
+                    f"Message to {wrapped_event.recipient} could not be sent: "
```

Comment:
slight nitpick here. This could be confusing as at this point in time the message has not even tried to be sent. To not get confused we should state that fetching metadata failed and that because of this `messages will most likely not be able to be sent`

Reply:
ok, I changed the message ✅